ui/app/components/clients/dashboard.js

```diff
@@ -45,7 +45,7 @@ export default class Dashboard extends Component {
 
   // filtering conditionals for HBS file
   // set initially based on response received from API & update based on filters
-  @tracked isDateRange;
+  @tracked isDateRange = true;
   @tracked isAllNamespaces; // false when filtered down to auth methods (mounts)
 
   get startTimeDisplay() {
```

Comment:
Does this need to be a getter? We could also use the `date-format` helper in HBS file
 `<p class="is-size-6">{{date-format this.startTime 'MMMM yyyy'}}</p>`

Reply:
Yes, because it modifies tracked properties. We can revisit if it needs to use the helper, but it handles the null situation which would need to be added to the helper and might not work for other instances where it is used.